spec/javascripts/govuk_publishing_components/modules.spec.js

```diff
@@ -154,12 +167,19 @@ describe('GOVUK Modules', function () {
     })
 
     it('does not start modules that are already started', function () {
-      var module = $('<div data-module="test-alert-frontend-module"></div>')
-      container.append(module)
+      var modules = $(
+        '<div data-module="test-alert-frontend-module"></div>' +
+        '<div data-module="test-alert-frontend-module-version-five"></div>' +
+        '<div data-module="gem-test-alert-frontend-module"></div>'
```

Comment:
This test looks to have changed from one test case to three. Just interested to know why this third one is in here - does the test need all three?

Reply:
I don't think the test needs all 3 to be honest, this could be reduced to testing 2 modules, those that have the `init` method and those that do not.

The reason for 3 different modules at the moment is to test, govuk-frontend frontend modules V4 and V5, and then modules from the gem.

Happy to change this and reduce down to 2 if that makes more sense?

Reply:
That explanation makes sense, thanks. Happy for you to leave as-is.